CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+cmake_minimum_required(VERSION 3.1)
+
+project(cli VERSION 1.0 LANGUAGES CXX)
+
+find_package(Boost 1.55 REQUIRED COMPONENTS system)
+find_package(Threads REQUIRED)
+
+add_library(cli INTERFACE)
```

Comment:
Should we use a variable for target name e.g. built-in`${PROJECT_NAME}`?

Reply:
I think it could be a good idea since the name of the project may change in the future (I personally think cli is way too generic).

But i also think the ultimate call must be made by @daniele77

Reply:
@tete17 I completely agree. "cli" is definitely too generic but -- you know -- it was an internal project and the name dates back to the days when I wouldn't imagine it would be used by others.

So:

1. @tiendq Ok, let's use a variable name for the target name
2. Let's think to a good name. Any idea? Something like "clipp" but not so abused. If we don't come up with a good descriptive name, I'll go for something completely uncorrelated to the library.
3. Do you know which are the issues in changing the name of a github project? The link, the repository name...? What kind of problems I will be facing?

Thanks, guys!